engine/src/main/java/org/pentaho/di/job/entries/copyfiles/JobEntryCopyFiles.java

```diff
@@ -93,6 +93,8 @@ public class JobEntryCopyFiles extends JobEntryBase implements Cloneable, JobEnt
   public static final String DEST_URL = "EMPTY_DEST_URL-";
   public static final String SOURCE_URL = "EMPTY_SOURCE_URL-";
 
+  private static final String XML_INDENTATION = "          ";
```

Comment:
added to get rid of critical sonar qube warning. The real issue this legacy code is hand writting xml, and addressing that is out of scope of this PR.